rag-and-text-to-sql/app.py

```
@@ -0,0 +1,264 @@
+        db = StructuredDataSQLHandler()
+        db.create_table(df)
+        db.insert_data(df)
+        text_to_sql_tool = Tool(
+            name="text_to_sql_tool",
+            description="A tool to query the structured document provided by the user. Call it to get understanding of comparative analysis of various Large Language Models (LLMs) based on key performance metrics like BLEU, ROUGE, Perplexity, Accuracy, Latencys. Argument required = user_question ",
+            tool_id="text_to_sql_001",
+            execute_fn= db.sql_tool 
+        )       
+        # **Initialize the chatbot tool**
+        chatbot_instance = OllamaLLM()  # Assuming CustomChatbot has `chatbot_tool` method
+        chatbotTool = Tool(
```

Ruff (0.8.2)

Local variable `chatbotTool` is assigned to but never used

Remove assignment to unused variable `chatbotTool`

(F841)